pkg/utils/helper.go

```diff
@@ -0,0 +1,74 @@
+package utils
```

Comment:
I agree with Antonio, the hashing stuff seems like it should be in hashing.go

Reply:
How I see it is to filter out the hash variables from the k8s object. That filtering out logic can be put into utils. I agree that filtering currently is specific to hashing. Keeping it in utils helps us to use it at other places if need arrives.

Reply:
Not really, because *what and how you filter out* is specific to `hashing` only. If we use this logic in other places, then these other places would have a dependency on *what parts of objects are used for hashing* and not on some *arbitrary* parts being filtered out (which is what name `utils` implies)

Reply:
Hmmm, I will move the helper class to hashing module and will still prefer to keep it out of hashing.go for better code clarity. What are your thoughts on this ?

Reply:
I do not see what helper class you speak of here.

Reply:
Kind of same class moved to hashing module.